hydroengine_service/main.py

```diff
@@ -1834,6 +1834,68 @@ def generate_image_info(im, params):
         'linearGradient': linear_gradient})
     return params
 
+def apply_image_operation(image, operation, data_params=None):
+    '''
+    Apply an operation to an image, based on specified operation and data parameters
+    :param image:
+    :param operation: String, type of operation
+    :param data_params: coming from data_visualization_parameters.json
+    :return:
+    '''
+    if operation == "log":
+        image = image.log10().rename('log')
+    if operation == "magnitude":
+        image = image.pow(2).reduce(ee.Reducer.sum()).sqrt().rename('magnitude')
+    if operation == "flowmap":
+        image.unitScale(data_params['min'][operation], data_params['max'][operation]).unmask(-9999)
+        data_mask = image.eq(-9999).select(data_params['bandNames'][band])
+        image = image.clamp(0, 1).addBands(data_mask).rename('flowmap')
+
+    return image
+
+@app.route('/get_feature_info', methods=['POST'])
+@flask_cors.cross_origin()
+def get_feature_info():
```

Comment:
Did you add a test for this in postman?

Reply:
2 tests- 1 for values returned, one for null value